modules/s3db/budget.py

```diff
@@ -1299,13 +1299,12 @@ def model(self):
                                           writable = True,
                                           represent = self.budget_CostItemRepresent(),
                                           ),
-                          # @ToDo: s3_datetime
-                          s3_date("start_date",
-                                  label = T("Start Date")
-                                  ),
-                          s3_date("end_date",
-                                  label = T("End Date")
-                                  ),
+                          s3_datetime("start_date",
```

Comment:
That doesn't explain why it wasn't done right away.
As I said, & Dominic has said before, if it was as trivial as you've done then it would have been done right away.
A ToDo is placed because itrequires more work than is currently avaialble to evaluate all the implications and come up with a full plan.